webui/react/src/pages/ExperimentDetails.table.tsx

```diff
@@ -30,9 +31,8 @@ export const columns: ColumnType<TrialItem>[] = [
   },
   {
     key: 'bestValidation',
-    render: (_: string, record: TrialItem): ReactNode => {
-      return record.bestValidationMetric ? humanReadableFloat(record.bestValidationMetric) : null;
-    },
+    render: (_: string, record: TrialItem): ReactNode =>
+      record.bestValidationMetric && humanReadableFloatRenderer(record.bestValidationMetric),
```

Comment:
**blocking:** The best and the latest metrics on the experiment detail page doesn't work for me...